### PR TITLE
Make Nette 3.0 compatible

### PR DESCRIPTION
**Pull request by TomasVotruba**

### CODE REVIEW

File: src/Kdyby/Annotations/DI/AnnotationsExtension.php [hunk 1]
```diff
@@ -81,7 +82,7 @@ public function getConfig(array $defaults = NULL, $expand = TRUE)
 			$config = Nette\DI\Config\Helpers::merge($config, ['ignore' => $globalConfig['doctrine']['ignoredAnnotations']]);
 		}
 
-		return $this->compiler->getContainerBuilder()->expand($config);
+		return Nette\DI\Helpers::expand($config, $this->getContainerBuilder()->parameters);
```
Comment (enumag) [inline src/Kdyby/Annotations/DI/AnnotationsExtension.php]: `Nette\DI\Helpers` is marked as internal. But probably necessary here for BC.

Comment (TomasVotruba) [inline src/Kdyby/Annotations/DI/AnnotationsExtension.php]: I didn't find any better solution yet.

Comment (enumag) [inline src/Kdyby/Annotations/DI/AnnotationsExtension.php]: Yeah. I doubt there is one. Expand shouldn't have been used here in the first place.

Comment (TomasVotruba) [inline src/Kdyby/Annotations/DI/AnnotationsExtension.php]: What should be used instead?

Comment (enumag) [inline src/Kdyby/Annotations/DI/AnnotationsExtension.php]: Constructor like in nette extensions.

Comment (TomasVotruba) [inline src/Kdyby/Annotations/DI/AnnotationsExtension.php]: I don't understand. Could you send PR with propper fix for this line?

Comment (enumag) [inline src/Kdyby/Annotations/DI/AnnotationsExtension.php]: As I said it's not worth fixing. It would be a BC break.

Comment (TomasVotruba) [inline src/Kdyby/Annotations/DI/AnnotationsExtension.php]: Not to apply it, but for me to understand it. I have no idea, what you mean.

I thought expand is used for `%appDir%` => absolute path conversion.

Comment (enumag) [inline src/Kdyby/Annotations/DI/AnnotationsExtension.php]: Indeed. But there is another way to do it now.

https://api.nette.org/2.3/source-Bridges.ApplicationDI.ApplicationExtension.php.html#40-48

```
extensions:
    nette.application: Nette\Bridges\ApplicationDI\ApplicationExtension( %debugMode% )
```

Comment (TomasVotruba) [inline src/Kdyby/Annotations/DI/AnnotationsExtension.php]: Thanks, that's it! Interesting, I will try using this.

Do you use this anywhere in your extensions?

Comment (enumag) [inline src/Kdyby/Annotations/DI/AnnotationsExtension.php]: Yes. I only rarely need parameters in my extensions but when I do I do it this way. For example https://github.com/Arachne/Twig/blob/master/src/DI/TwigExtension.php#L48.

Comment (TomasVotruba) [inline src/Kdyby/Annotations/DI/AnnotationsExtension.php]: Awesome, thanks'

**Pull request closed without merging**